### DISCUSSION TITLE
History

### DISCUSSION THREAD

**Discussion by arcimus** [LeoKlaus/plappa#14] [Feature Requests]
Hello.  Thanks for your efforts on plappa.  The feature I find myself missing the most since moving from Plex/Prologue is Prologue's history feature:
![image](https://github.com/LeoKlaus/plappa/assets/34137090/311d2f25-6d24-498c-a7ca-8691104d68d9)

**Comment by LeoKlaus** [comment]
Hey arcimus, thank you for the suggestion!

Currently, this information is not tracked by plappa so it would require some restructuring of the progress handling but I think it's a nice touch. I'll definitely consider implementing this!

**Comment by LeoKlaus** [comment]
I've added this with a recent update (IIRC it's TestFlight only for now), will release with the next AppStore update!

**Reply by Binocularbath** [reply]
Hi Leo

I'm trying to find an app for my girlfriend so she can access the audiobookshelf server from her iOS device. I've got it running on plappa. Seems great. I'll probably buy it so she can have all features. I wanted to ask two questions 

1) have this feature been implemented? The listening history 
2) is there a plan to go fully open source with the code?
